test/integration/02-user-wallet/02-send-lightning.spec.ts

```diff
@@ -590,12 +591,19 @@ describe("UserWallet - Lightning Pay", () => {
           }
         })
 
-        const userWallet1Path = userWallet1.user.accountPath
-        const query = { type: "payment", pending: true, voided: false }
```

Comment:
I assume the `voided: false` here is redundant because in the new implementation we only void payments after settling (setting `pending: false`). All tests pass.